src/css/base.scss

```diff
@@ -164,14 +197,31 @@ h5 {
 }
 
 .travel-icon {
-  width: 50%;
+  width: 80%;
   position: relative;
+  margin-top: 5%;
+  margin-left: 10%;
+  margin-bottom: 10%;
+}
+
+.traveler-icon {
+  width: 10%;
 }
 
 .tracker-heading {
-  font-size: xx-large;
+  font-size: 40px;
+  text-decoration: underline;
 }
 
 .money-spent-text {
   margin: 0px;
-}
\ No newline at end of file
+}
+
+.hidden {
+  display: none!important;
```

Comment:
What do you think about having display:none instead? It has the same functionality.